pkg/orchestration/wiring/datadog/types.go

```diff
@@ -0,0 +1,57 @@
+package datadog
+
+import (
+	"github.com/atlassian/voyager"
+)
+
+type AlarmSpecType string
+type AlarmType string
+
+const (
+	Metric AlarmSpecType = "metric alert"
+)
+const (
+	CPU    AlarmType = "cpu"
+	Memory AlarmType = "memory"
+)
+
+type Alarm struct {
+	Name    string      `json:"name"`
+	Type    string      `json:"type"`
+	Query   string      `json:"query"`
+	Option  AlarmOption `json:"options"`
+	Message string      `json:"message"`
+}
+
+type ServiceInstanceSpec struct {
```

Comment:
This is a bad name for type (I see that it's the same in OAP, but still), because there is already ServiceInstanceSpec type in Service Catalog API, and this struct will be marshalled into Service Catalog's `ServiceInstanceSpec.Parameters.Raw`.

I propose to rename it to just `InstanceParameters` or `OSBInstanceParameters`.